docs/ref/models/fields.txt

```diff
@@ -1184,18 +1184,21 @@ values are stored as null.
 
 .. versionadded:: 3.1
 
-A field for storing JSON encoded data. In Python the data is represented in
-its Python native format: dictionaries, lists, strings, numbers, booleans
-and ``None``.
+A field for storing JSON encoded data. In Python the data is represented in its
+Python native format: dictionaries, lists, strings, numbers, booleans and
+``None``.
+
+``JSONField`` is supported on MariaDB 10.2.7+, MySQL 5.7.8+, Oracle,
+PostgreSQL, and SQLite 3.9.0+ (with the `JSON1 extension`_ enabled).
 
 .. attribute:: JSONField.encoder
 
     An optional :py:class:`json.JSONEncoder` subclass to serialize data types
-    not supported by the standard JSON serializer (``datetime``, ``uuid``,
-    etc.). For example, you can use the
+    not supported by the standard JSON serializer (``datetime.datetime``,
+    :class:`~python:uuid.UUID`, etc.). For example, you can use the
```

Comment:
@felixxm 
You changed

```
(``datetime.datetime``, :class:`~python:uuid.UUID`, etc.).
```

into

```
(e.g. ``datetime.datetime`` or :class:`~python:uuid.UUID`).
```

in `docs/ref/forms/fields.txt`, just thought that maybe you want to do the same here.